modules/base/processors/volumeslice.cpp

```diff
@@ -168,15 +165,19 @@ void VolumeSlice::process() {
                 auto layerdata = layerrep->getDataTyped();
 
                 switch (util::extent<T, 0>::value) {
```

Comment:
extent<char>::value (and other non vec types) return zero wile for vecX it return X as expected: Hence ppl expect it to return 1 for scalars